packages/blade/src/components/Button/BaseButton/__tests__/BaseButton.native.test.tsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.native';
+import BaseButton from '../BaseButton';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<BaseButton />', () => {
+  it('should render button with default properties', () => {
+    const buttonText = 'Pay Now';
+    const { toJSON, getByText } = renderWithTheme(<BaseButton>{buttonText}</BaseButton>);
+    expect(toJSON()).toMatchSnapshot();
+    expect(getByText('Pay Now')).toBeTruthy();
+  });
+  it('should throw an error when there is no icon or text passed to the button', () => {
+    try {
+      // @ts-expect-error
```

Comment:
can you mention the description about the expected error here? or is it temporary and will be handled in subsequent PRs?

Reply:
bumping this